src/operators.jl

```diff
@@ -858,3 +855,26 @@ function merge_vertices!(g::Graph{T}, vs::Vector{U} where U <: Integer) where T
 
     return new_vertex_ids
 end
+
+# special case for digraphs
+function merge_vertices!(g::AbstractGraph, vs::Vector{U} where U <: Integer)
+    @assert is_directed(g)
+    vs = sort(vs, rev=true)
```

Comment:
is there a reason to modify the input vector? If so, then use `sort!`.

Reply:
Good suggestion; it's probably better that we don't edit the input vector.